### PR TITLE
update apis for new my evaluation results screen

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [My Evaluaton Results - Enhancements Revision 1#368](https://github.com/nerubia/kh360-react/issues/368)

FE PR: [update my evaluation results screen for enhancements #382](https://github.com/nerubia/kh360-react/pull/382)

### CODE REVIEW

**Review by jatnerubia — APPROVED** [review]
Cool

File: src/controllers/user/score-rating-controller.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { type Request, type Response } from "express"
+import * as ScoreRatingSerice from "../../services/score-rating-service"
```
Comment (jatnerubia) [inline src/controllers/user/score-rating-controller.ts:2]: typo

File: src/routes/user/user-route.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import express from "express"
 import * as EmailTemplateController from "../../controllers/user/email-template-controller"
+import * as ScoreRatingController from "../../controllers/user/score-rating-controller"
 import * as UserController from "../../controllers/user/user-controller"
 import * as LogoutController from "../../controllers/user/logout-controller"
 
 const router = express.Router()
 
 router.get("/email-templates", EmailTemplateController.index)
+router.get("/score-ratings", ScoreRatingController.index)
```
Comment (jatnerubia) [inline src/routes/user/user-route.ts:10]: fix order

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Comment by emmaacha** [comment]
Done ^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by gborgonia-ir — APPROVED** [review]

**Pull request closed without merging**